server/models/ltiUser.ts

```diff
@@ -7,6 +7,7 @@ export const LtiUserSchema = {
   properties: {
     id: { title: "End-User ID", type: "string" },
     name: { title: "End-User's full name in displayable", type: "string" },
+    email: { title: "End-User's email in displayable", type: "string" },
```

Comment:
```suggestion
    email: { title: "End-User's preferred e-mail address", type: "string" },
```

http://www.imsglobal.org/spec/lti/v1p3/#:~:text=email%3A%20per%20oidc%20specifcations%2C%20end-user's%20preferred%20e-mail%20address.